tools/tests/com.liferay.ide.gradle.core.tests/src/com/liferay/ide/gradle/core/tests/GradleScriptASTVisitor.java

```diff
@@ -0,0 +1,36 @@
+package com.liferay.ide.gradle.core.tests;
+
+
+import org.codehaus.groovy.ast.CodeVisitorSupport;
+import org.codehaus.groovy.ast.expr.MethodCallExpression;
+
+/**
+ * @author Lovett Li
+ */
+public class GradleScriptASTVisitor extends CodeVisitorSupport
```

Comment:
Name this class similar to its function.  This this class will only find the correct location of dependencies insertion.  Lets name that class accordinly, something FindDependenciesBlockInsertionPoint or something